Packages/vcs/vcs/vector.py

```diff
@@ -528,6 +535,26 @@ def _setalignment(self, value):
         self._alignment = value
     alignment = property(_getalignment, _setalignment)
 
+
```

Comment:
@aashish24 I would rather the set/get function to be in VCS_Validation functions, just in case we ever need to re-use it for something else and have here
```
scalerange = VCS_validation_functions.scalerange
```
see colormap above for example.